modules/gitops-release-notes-1-5-5.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-5-5_{context}"]
+= Release notes for {gitops-title} 1.5.5
+
+{gitops-title} 1.5.5 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="new-features-1-5-5_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.3.7.
+
+[id="fixed-issues-1-5-5_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCC's were in place in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]
+
+[id="known-issues-1-5-5_{context}"]
+== Known issues
+
+*  {gitops-title} Operator can make use of RHSSO (KeyCloak) through OIDC in addition to Dex. However, with a recent security fix applied, the certificate of RHSSO cannot be validated in some scenarios. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
++
+As a workaround, disable TLS validation for the OIDC (Keycloak/RHSSO) endpoint in the ArgoCD specification. 
+
+[source,yaml]
+----
+spec:
```

Comment:
Please add the following information, as indicated in
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields:
- Include the apiVersion and kind so that a user always knows the context of the YAML.
- Include the full hierarchy to a deeply nested key.

Reply:
And do the same for the 1-4 file.